src/Components/Reviews.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from "react";
+import { fetchAllReviews } from "../api";
+import ReviewCard from "./ReviewCard";
+import "./style/Reviews.css";
+
+const Reviews = () => {
+  const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetchAllReviews().then((data) => {
+      setReviews(() => {
+        return [...data];
+      });
```

Comment:
you can just set reviews to be data here without having to spread into a new array